### PR TITLE
adds a prompt to inform users about stuff

### PR DESCRIPTION
**Pull request by PhilipJFryFAF**
replaces the tutorial prompt with info about the fact that the original campaign missions are currently broken and some info about how to play the faf patched campaign

### CODE REVIEW

**Comment by speed2CZ** [comment]
You could add a button to open the wiki page when you're mentioning it there

**Comment by PhilipJFryFAF** [comment]
i didn't even know that we can open an url from within the game
is there an example of that being done somewhere?

File: lua/ui/menus/main.lua [hunk 1]
```diff
@@ -799,8 +799,14 @@ function CreateUI()
         end
     end
 
+    function CampaignPrompt(callback)
+      UIUtil.QuickDialog(GetFrame(0), "The Campaign missions from the base game are broken! \n Please use the coop maps from the FAForever client or start the game offline with the coop init file to play the FAF patched campaign. For more information visit wiki.faforever.com",
+      "<LOC _Ok>", callback()
```
Comment (speed2CZ) [inline lua/ui/menus/main.lua]: for this line try:
```LUA
"<LOC _Ok>", callback(), "Open Wiki", function() OpenURL('www.wiki.faforever.com') end```
with correct link

**Comment by speed2CZ** [comment]
```LUA
    function CampaignPrompt(callback)
        UIUtil.QuickDialog(GetFrame(0), "The Campaign missions from the base game are broken! \n Please use the coop maps from the FAForever client or start the game offline with the coop init file to play the FAF patched campaign. For more information visit wiki.faforever.com",
        "<LOC _Ok>", callback(),
        "Open Wiki", function()
            OpenURL('http://wiki.faforever.com/index.php?title=Mission_Scripting')
        end
    )
    end
```
the URL string has to start with `http` NOT with `https`

**Comment by Sheeo** [comment]
Why not disable the button instead and give it a tooltip with the same explanation?

**Comment by PhilipJFryFAF** [comment]
sry for the late response - i should probably enable email notification...
if we disable that menu completely then ppl can't watch the video explanations and stuff

**Pull request closed without merging**